spec/card_spec.rb

```diff
@@ -10,7 +11,7 @@
 
   it "has readable attributes" do
     card = Card.new(:diamond, 'Queen', 12)
-
+    #binding.pry 
```

Comment:
It's generally best practice to remove unused/commented out code before we make a PR. We don't want to merge unused code to main, even if it's just a test.